test/plots/piePlotTests.ts

```diff
@@ -2,92 +2,140 @@
-      assert.lengthOf(twoSlicePathStrings, twoSliceData.length * 2, "2 paths per datum");
-
-      twoSlicePathStrings.forEach((pathString, index) => {
-        assert.notStrictEqual(pathString, fourSlicePathStrings[index], "slices were updated when data changed");
+        let secondPathPoints1 = pathPoints1[2].split(",");
+        assert.closeTo(parseFloat(secondPathPoints1[0]), 0, 1, "draws line to origin");
+        assert.closeTo(parseFloat(secondPathPoints1[1]), 0, 1, "draws line to origin");
+        svg.remove();
+      });
+
+      it("uses a plottable-colors color scale to color sectors by default", () => {
+        let arcPaths = piePlot.content().selectAll(".arc");
+
+        let arcPath0 = d3.select(arcPaths[0][0]);
+        assert.strictEqual(arcPath0.attr("fill"), "#5279c7", "first sector filled appropriately");
+
+        let arcPath1 = d3.select(arcPaths[0][1]);
+        assert.strictEqual(arcPath1.attr("fill"), "#fd373e", "second sector filled appropriately");
+        svg.remove();
+      });
+
+      it("updates slices when data changes", () => {
+        let originalPathStrings: String[] = [];
+        piePlot.content().selectAll("path").each(function() { originalPathStrings.push(d3.select(this).attr("d")); });
+        assert.lengthOf(originalPathStrings, simpleData.length * 2, "2 paths per datum");
+
+        let oneMoreSliceData = simpleData.slice();
+        oneMoreSliceData.push({ value: 5 });
+        simpleDataset.data(oneMoreSliceData);
+
+        let oneMoreSlicePathStrings: String[] = [];
+        piePlot.content().selectAll("path").each(function() { oneMoreSlicePathStrings.push(d3.select(this).attr("d")); });
+        assert.lengthOf(oneMoreSlicePathStrings, oneMoreSliceData.length * 2, "2 paths per datum");
+
+        originalPathStrings.forEach((pathString, index) => {
+          assert.notStrictEqual(pathString, oneMoreSlicePathStrings[index], "slices were updated when data changed");
+        });
+        svg.remove();
+      });
+
+      it("can set innerRadius", () => {
+        let expectedInnerRadius = 5;
+        piePlot.innerRadius(expectedInnerRadius);
+        let arcPaths = piePlot.content().selectAll(".arc.fill");
+        assert.lengthOf(arcPaths[0], 2, "only has two sectors");
+
+        let pathPoints0 = TestMethods.normalizePath(d3.select(arcPaths[0][0]).attr("d")).split(/[A-Z]/).slice(1, 5);
+
+        let radiusPath0 = pathPoints0[2].split(",").map((coordinate: string) => parseFloat(coordinate));
+        assert.closeTo(radiusPath0[0], expectedInnerRadius, 1, "stops line at innerRadius point");
+        assert.closeTo(radiusPath0[1], 0, 1, "stops line at innerRadius point");
+
+        let innerArcPath0 = pathPoints0[3].split(",").map((coordinate: string) => parseFloat(coordinate));
+        assert.closeTo(innerArcPath0[0], expectedInnerRadius, 1, "makes inner arc of correct radius");
+        assert.closeTo(innerArcPath0[1], expectedInnerRadius, 1, "makes inner arc of correct radius");
```

Comment:
The two asserts have identical messages so it is hard to figure out which one failed.